pkg/action/install_test.go

```diff
@@ -196,131 +182,131 @@ func TestInstallAllSharedDeps(t *testing.T) {
-			dep: &chart.Dependency{
-				Dependency: &helmChart.Dependency{
-					Name:       "testdata/charts/shared-dep",
-					Version:    "~0.1.0",
-					Repository: "",
-				},
-				IsOptional: false,
-			},
-			status:  "deployed",
-			ns:      "my-shared-dep-ns",
-			relName: "my-shared-dep",
-		},
-		{
-			name: "install non-existent dep",
-			dep: &chart.Dependency{
-				Dependency: &helmChart.Dependency{
-					Name:       "nonexistent-chart",
-					Version:    "0.1.0",
-					Repository: "",
-				},
-				IsOptional: false,
-			},
-			wantError: true,
-			error:     "failed to download \"nonexistent-chart\" (hint: running `helm repo update` may help)",
-		},
-		{
-			name: "shared-dep version cannot be found",
-			dep: &chart.Dependency{
-				Dependency: &helmChart.Dependency{
-					Name:       "testdata/charts/shared-dep",
-					Version:    "1.1.0",
-					Repository: "",
-				},
-				IsOptional: false,
-			},
-			wantError: true,
-			error:     "Satisfiable chart version not found; 0.1.0 is not equal to 1.1.0",
-		},
-		{
-			name: "shared-dep version non-parseable",
-			dep: &chart.Dependency{
-				Dependency: &helmChart.Dependency{
-					Name:       "testdata/charts/shared-dep",
-					Version:    "foo0.1.0",
-					Repository: "",
-				},
-				IsOptional: false,
-			},
-			wantError: true,
-			error:     "improper constraint: foo0.1.0",
-		},
-	} {
-		is := assert.New(t)
-
-		// create our own Logger that satisfies impl/cli.Logger, but with a buffer for tests
-		buf := new(bytes.Buffer)
-		logger := logcli.NewStandard()
-		logger.InfoOut = buf
-		logger.WarnOut = buf
-		logger.ErrorOut = buf
-		logger.DebugOut = buf
-		log.Current = logger
-
-		settings := cli.New()
-
-		instAction := installAction(t)
-
-		instAction.DryRun = tcase.wantDryRun
-
-		res, err := instAction.InstallSharedDep(tcase.dep, tcase.ns, settings, log.Current, 0)
-		if (err != nil) != tcase.wantError {
-			t.Errorf("on test %q expected error, got '%v'", tcase.name, err)
-		}
-		if tcase.wantError {
-			is.Equal(tcase.error, err.Error())
-		} else {
-			is.Equal(res.Info.Status.String(), tcase.status, "Expected status of the installed dependency.")
-			if tcase.status == "deployed" {
-				is.Equal(res.Name, tcase.relName, "Expected release name from dependency.")
-				is.Equal(res.Namespace, tcase.ns, "Expected parent ns.")
-			}
-		}
-	}
-}
+// func TestInstallSharedDep(t *testing.T) {
+
+// 	for _, tcase := range []struct {
+// 		name       string
+// 		dep        *chart.Dependency
+// 		wantError  bool
+// 		error      string
+// 		wantDryRun bool
+// 		status     string
+// 		relName    string
+// 		ns         string
+// 	}{
+// 		{
+// 			name: "dry-run-is-passed",
+// 			dep: &chart.Dependency{
+// 				Dependency: &helmChart.Dependency{
+// 					Name:       "testdata/charts/vanilla-helm",
+// 					Version:    "0.1.0",
+// 					Repository: "",
+// 				},
+// 				IsOptional: false,
+// 			},
+// 			wantDryRun: true,
+// 			status:     "pending-install",
+// 		},
```

Comment:
Will these tests be setup to work again?

Reply:
Yes, these are the non-working ones we were talking about today.

Reply:
done, see previous comment.